testCommon/src/test/java/com/virtuslab/gitmachete/testcommon/BaseGitRepositoryBackedIntegrationTestSuite.java

```diff
@@ -13,12 +13,12 @@
 
 public abstract class BaseGitRepositoryBackedIntegrationTestSuite {
 
-  protected final static String SETUP_FOR_NO_REMOTES = "setup-with-no-remotes.sh";
-  protected final static String SETUP_WITH_SINGLE_REMOTE = "setup-with-single-remote.sh";
-  protected final static String SETUP_WITH_MULTIPLE_REMOTES = "setup-with-multiple-remotes.sh";
-  protected final static String SETUP_FOR_DIVERGED_AND_OLDER_THAN = "setup-for-diverged-and-older-than.sh";
-  protected final static String SETUP_FOR_YELLOW_EDGES = "setup-for-yellow-edges.sh";
-  protected final static String SETUP_FOR_OVERRIDDEN_FORK_POINT = "setup-for-overridden-fork-point.sh";
+  public final static String SETUP_FOR_NO_REMOTES = "setup-with-no-remotes.sh";
```

Comment:
Hmm do they need to be public actually?

Reply:
it is actually so that it can be used like this:
`import ...SETUP_WITH_SINGLE_REMOTE
`
`class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH_SINGLE_REMOTE) {
`

Reply:
Aaaaa ok, Scala's primary c'tor, right... ok, let's leave as is